### DISCUSSION TITLE
【求助】作者大大能不能加一个查看我的帖子的功能，我根本找不到自己发的帖子

### DISCUSSION THREAD

**Discussion by did1181** [share121/inter-knot#2905] [其他]
![屏幕截图 2024-08-01 181856](https://github.com/user-attachments/assets/79fdfa23-140b-47cb-be6c-b0432078b841)
我发的帖子 全到后面去了，想看个回复都难

**Comment by 6078HITHEDAY** [comment]
github评论筛选用户

**Reply by did1181** [reply]
此话怎讲？

**Reply by 6078HITHEDAY** [reply]
> 此话怎讲？

![Screenshot_20240819-004735.png](https://github.com/user-attachments/assets/b91631a2-be1d-4565-868b-6bb7e9320d43)

github~app

**Comment by HKLHaoBin** [comment]
### 这里
![Clip_2024-08-19_00-45-46](https://github.com/user-attachments/assets/0d21c72b-b11b-47d0-85f6-ab36fa0acaf5)

<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by did1181** [comment]
感谢帮助，只是我想在绳网内回复，在GitHub上回复没那种感觉
![Uploading image.png…]()

**Reply by 6078HITHEDAY** [reply]
那你开发一下
